AstraEngine.Core/Component.cs

```diff
@@ -13,6 +13,8 @@ public abstract class Component
     /// </summary>
     [AllowNull]
     public Entity Entity { get; internal set; }
+    /// <summary>Whether or not this <see cref="Component"/> is active</summary>
+    public bool Active { get; set; } = true;

```

Comment:
Boolean naming convention says to use affirmative phrases e.g. `IsActive`. See: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members